simphony/cuds/abc_dataset.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+from abc import ABCMeta, abstractmethod
+
+
+class ABCDataset(object):
```

Comment:
I think the name is fine. How about adding `meta.DataSet` as a parent to get the properties?